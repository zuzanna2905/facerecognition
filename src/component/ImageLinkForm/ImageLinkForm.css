.form{
    width: 700px;
    background-image: url("./form.png");
}