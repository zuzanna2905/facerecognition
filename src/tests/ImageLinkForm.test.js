import {shallow} from 'enzyme';
import React from 'react';
import ImageLinkForm from '../component/ImageLinkForm/ImageLinkForm';

it('renders ImageLinkForm without crashing', () => {
    let  wrapper = shallow(<ImageLinkForm/>);
    expect(wrapper).toMatchSnapshot();
});