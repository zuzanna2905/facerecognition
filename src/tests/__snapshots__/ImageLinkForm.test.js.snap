// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageLinkForm without crashing 1`] = `
<div>
  <p
    className="f3"
  >
    This Magic Brain will detect faces in your pictures.
  </p>
  <div
    className="center"
  >
    <div
      className="form center pa4 br3 shadow-5"
    >
      <input
        className="f4 pa2 w-70 center"
        type="text"
      />
      <button
        className="w-30 grow f4 link ph3 pv2 dib white bg-light-pink"
      >
        Detect
      </button>
    </div>
  </div>
</div>
`;
